protos/followme/followme.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package mavsdk.rpc.followme;
+
+option java_package = "io.mavsdk.followme";
+option java_outer_classname = "FollowMeProto";
+
+service FollowMeService {
+  /*
+   * Gets current FollowMe configuration.
+   */
+  rpc GetConfig(GetConfigRequest) returns(GetConfigResponse) {}
+  /*
+   * Applies FollowMe configuration by sending it to the system.
+   */
+  rpc SetConfig(SetConfigRequest) returns(GetConfigResponse) {}
+  /*
+   * Checks if FollowMe is active.
+   */
+  rpc IsActive(IsActiveRequest) returns(IsActiveResponse) {}
+  /*
+   * Sets location of the moving target.
+   */
+  rpc SetTargetLocation(SetTargetLocationRequest) returns(SetTargetLocationResponse) {}
+  /*
+   * Gets the last location of the target.
+   */
+  rpc GetLastLocation(GetLastLocationRequest) returns(GetLastLocationResponse) {}
+  /*
+   * Starts FollowMe mode.
+   */
+  rpc Start(StartRequest) returns(StartResponse) {}
+  /*
+   * Stops FollowMe mode.
+   */
+  rpc Stop(StopRequest) returns(StopResponse) {}
+}
+
+
+
+// FollowMe Config type.
+message Config {
```

Comment:
This message is missing some documentation. [This one](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L273-L284) is a good similar example.